src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { filterSlice } from '../../features/filter';
+import { useAppSelector } from '../../app/hooks';
 
 export const TodoFilter: React.FC = () => {
+  const { query, status } = useAppSelector(state => state.filter);
+
+  const dispatch = useDispatch();
+
+  const handleStatus = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    dispatch(filterSlice.actions.setStatus(event.target.value));
```

Comment:
It's better to export actions from the slice file and import them here, in slice:

```
export const filterActions = filterSlice.actions;
```
or even 
```
export const { setStatus, setQuery } = filterSlice.actions;
```